platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/cleanup/ProgramExecutionCleanupProgram.java

```diff
@@ -0,0 +1,26 @@
+package com.softicar.platform.core.module.program.execution.cleanup;
+
+import com.softicar.platform.core.module.program.IProgram;
+import com.softicar.platform.core.module.program.execution.AGProgramExecution;
+import com.softicar.platform.core.module.program.execution.AGProgramExecutionLog;
+import com.softicar.platform.emf.source.code.reference.point.EmfSourceCodeReferencePointUuid;
+
+/**
+ * Deletes program execution records ({@link AGProgramExecution}) and their log
```

Comment:
- We could remove some redundancy here, like: `Deletes ({@link AGProgramExecution}) records and corresponding ({@link AGProgramExecutionLog}) records [...]`
- Also, there's `[...] records records [...]` in the Javadoc.

Reply:
done